src/config/ScenarioParameters.js

```diff
@@ -64,6 +64,66 @@ const PARAMETERS = {
     dataCy: 'start_date',
     defaultValue: new Date('2014-08-18T21:11:54'),
   },
+  additional_seats: {
+    dataCy: 'additional_seats',
+    defaultValue: -4,
+    minValue: -600,
+    maxValue: 2500,
+    labels: {
+      en: 'Additional seats (min -600, max 2500)',
+      fr: 'Sièges addtionnels (min -600, max 2500)',
+    },
+  },
+  activated: {
+    dataCy: 'activated',
+    defaultValue: false,
+  },
+  additional_tables: {
+    dataCy: 'additional_tables',
+    defaultValue: 3,
+    minValue: -150,
+    maxValue: 12000,
+    labels: {
+      en: 'Additional seats (min -150, max 12000)',
+      fr: 'Sièges addtionnels (min -150, max 12000)',
+    },
+  },
+  volume_unit: {
+    dataCy: 'volume_unit',
+    defaultValue: 'LITRE',
+    labels: {
+      en: 'Volume unit',
+      fr: 'Unité de volume',
+    },
+    enumValues: [
+      {
+        key: 'LITRE',
+        value: 'L',
+      },
+      {
+        key: 'BARREL',
+        value: 'bl',
+      },
+      {
+        key: 'CUBIC_METRE',
+        value: 'm³',
+      },
+    ],
+  },
+  evaluation: {
+    dataCy: 'evaluation',
+    defaultValue: 'Good',
+  },
+  comment: {
+    dataCy: 'comment',
+    defaultValue: 'None',
+  },
+  additional_date: {
+    dataCy: 'additional_date',
+    defaultValue: new Date(),
+    minValue: new Date('2022/01/01'),
+    maxValue: new Date('2022/12/31'),
```

Comment:
Default value will become invalid next year... can we try to make min & max values relative to the default value (e.g. current year + 1) or hard-code the default value ?